core-rust/mesh-api-server/src/mesh_api/conversions/operations.rs

```diff
@@ -140,3 +186,129 @@ fn resolve_global_fee_balance_changes(
     }
     Ok(fee_balance_changes)
 }
+
+/// This method converts Transaction V1 instructons to operations.
+/// Note that it supports very limited number of instructions because it
+/// is implemented just for sanity checks.
+pub fn to_mesh_api_operations_from_instructions_v1(
+    instructions: &[InstructionV1],
+    mapping_context: &MappingContext,
+    database: &StateManagerDatabase<impl ReadableRocks>,
```

Comment:
As per a few files, we tend to use `&ActualStateManagerDatabase` here instead.

Reply:
Sounds good. I was inspired by Core API, which mainly uses  `&StateManagerDatabase<impl ReadableRocks>`. Maybe, as a followup, it would make sense to refactor Core API as well.